package.json

```diff
@@ -29,92 +30,40 @@
     ]
   },
   "scripts": {
-    "start": "tsc -w",
-    "build": "yarn clean && yarn tsc && yarn copy-templates && yarn copy-types",
-    "copy-types": "copyfiles -u 1 ./src/*.d.ts ./dist",
-    "copy-templates": "copyfiles -u 1 ./src/routeGeneration/templates/**/* ./dist",
-    "clean": "rimraf dist && rimraf tests/fixtures/*/routes.ts",
-    "lint": "tslint ./src/**/*.ts ./tests/**/*.ts",
-    "format": "tsfmt -r",
+    "build": "lerna run build --stream",
+    "deploy": "yarn bump && lerna publish from-git",
+    "bump": "lerna version patch -m \"Release v%s\" --force-publish",
+    "clean": "lerna run clean --stream",
     "prepare": "yarn build",
-    "deploy": "yarn version patch -m \"Release v%s\" && yarn publish",
     "preversion": "yarn test",
-    "postversion": "git push origin master && git push --follow-tags",
-    "pretest": "cross-env NODE_ENV=tsoa_test ts-node ./tests/prepare.ts",
-    "loading-msg-for-tests": "echo '... starting a test run to ensure quality. Note this takes some time as tests do not run in parallel (by design) ...' ",
-    "test": "node ensureYarn.js && tsc -p ./tests/tsconfig.json --noEmit && yarn loading-msg-for-tests && cross-env NODE_ENV=tsoa_test mocha **/*.spec.ts --exit --require ts-node/register --timeout 5000",
+    "test": "yarn --cwd tests test",
+    "lint": "lerna run lint --stream",
```

Comment:
Because all sub projects has same linting rules maybe it is better to run them from root config, and remove that logic from sub configs, e.g. in root `package.json`:
```    
"lint": "eslint packages/**/*.{ts,tsx}",
```
I agree that build flow should be unique per project, but looks like that linting it is pretty common thing.

Reply:
Yeah I agree, I've moved this now